docs/README.md

```diff
@@ -178,25 +178,25 @@
     * [Health Check Library](extensions/healthcheck-library.md)
   * [CA Rotation in Extensions](extensions/ca-rotation.md)
   * Blob storage providers
-    * [`BackupBucket` resource](extensions/backupbucket.md)
-    * [`BackupEntry` resource](extensions/backupentry.md)
+    * [`BackupBucket` resource](extensions/resources/backupbucket.md)
+    * [`BackupEntry` resource](extensions/resources/backupentry.md)
   * DNS providers
     * [`DNSRecord` resources](extensions/dnsrecord.md)
   * IaaS/Cloud providers
     * [Control plane customization webhooks](extensions/controlplane-webhooks.md)
-    * [`Bastion` resource](extensions/bastion.md)
-    * [`ControlPlane` resource](extensions/controlplane.md)
-    * [`ControlPlane` exposure resource](extensions/controlplane-exposure.md)
-    * [`Infrastructure` resource](extensions/infrastructure.md)
-    * [`Worker` resource](extensions/worker.md)
+    * [`Bastion` resource](extensions/resources/bastion.md)
```

Comment:
Most of the extension resources have a title. Example: https://github.com/gardener/gardener/blob/6e98ed02183da06434f957be724b24cb1d9b2e62/docs/extensions/resources/backupbucket.md#L1-L3

But `docs/extensions/resources/infrastructure.md`, `docs/extensions/resources/bastion.md`, `docs/extensions/resources/extension.md`, `docs/extensions/resources/network.md`, `docs/extensions/resources/operatingsystemconfig.md` and `docs/extensions/resources/worker.md` don't have such title. 

Please unify and make it consistent everywhere!

Reply:
The comment not being resolved could be an issue due to the rebase. I will go fix it.

Reply:
As for the titles, the ones that have a specific title set in the frontmatter have a unique capitalization that can't be reflected in their file name. If a topic does not have a title set in its frontmatter, its title will be its file name but capitalized -> `bastion.md` = Bastion

Therefore, for such topics it isn't needed to set a title in the frontmatter. I will set the title for `operatingsystemconfig.md`, though, as the title there should be OperatingSystemConfig